### DISCUSSION TITLE
How can I check browser support?

### DISCUSSION THREAD

**Discussion by keithlayne** [preactjs/preact#3744] [Q&A] [answered]
I am considering using preact for a project that will run on devices with a browser based of a fairly ancient chromium. I just want to establish if preact is feasible in this case - it has a lot of appeal since we're familiar with react but (AFAIK) can't use recent react on this project and don't really want all the overhead.

I saw https://github.com/preactjs/preact-www/issues/108 but I have no clue what JS language features and DOM APIs current versions of preact require. I'm mostly worried about anything that couldn't be polyfilled. Polyfills are probably okay but I'm not worried about that really right now.

I haven't dug into preact, and please forgive me if I've just failed to RTFM. If I can do something simple like load a browser-based test suite in the target environment, that would be terrific.

**Comment by JoviDeCroock** [comment, marked as answer]
Current Preact X supports all modern browsers and IE11 so you shouldn't run into any issues, in Preact 11 we are dropping IE11

**Reply by keithlayne** [reply]
Thanks! I saw your PR to drop IE11. I think mainly I was unsure of what "modern browsers" meant. I'm talking about chrome 45, not like chrome 3 or anything. Mostly worried about requiring ESM or Proxy and stuff like that.

**Reply by JoviDeCroock** [reply]
None of those are used and we ship with umd/esm/cjs

**Reply by marvinhagemeister** [reply]
Like @JoviDeCroock said, we support all browsers down to IE11 with Preact X. There was some talk about upping browser support, but that would definitely be a breaking change and therefore would only happen with a major release (v11).
